rllib/algorithms/algorithm_config.py

```diff
@@ -1343,6 +1343,14 @@ def training(
                     error=True,
                 )
             self.model.update(model)
+            if model.get("_use_default_native_models"):
+                Deprecated(
```

Comment:
Let's change this to `deprecation_warning` (`ray.rllib.utils.deprecation::deprecation_warning(...)`)